docs/content/preview/troubleshoot/nodes/disk-full.md

```diff
@@ -1,25 +1,136 @@
+
+- Scale up
+
+   Switch to bigger machines or add more storage disks. If you are using YugabyteDB Anywhere, follow the instructions in [Expanding universe node disk capacity with YugabyteDB Anywhere](https://support.yugabyte.com/hc/en-us/articles/5463616207757-Expanding-universe-node-disk-capacity-with-YugabyteDB-Anywhere-platform).
+
+- Scale out
+
+    Add more nodes to the cluster. The YugabyteDB load balancer will rebalance the tablets and the data across the added nodes.
+
+### Remove unnecessary files from the nodes
+
+Unnecessary files can accumulate on the nodes, including large log files or older core dumps.
+
+Do not modify or remove the YugabyteDB data, bin, or config directories and files. Damage or loss of these files can result in unavailability and data loss.
+
+### Drop unnecessary tables and databases/namespaces
+
+Dropping large tables, YSQL databases, or YCQL namespaces that are no longer necessary will help recover disk space.
+
+When dropping tables on a database/namespace with [Point-in-time recovery](../../../manage/backup-restore/point-in-time-recovery/) enabled, the tables are set to a HIDDEN state and won't be cleaned up until all their snapshot schedules have expired.
+
+When dropping YSQL tables, ensure the YB-Master has enough disk space, as this involves updating the YSQL catalog tables.
+
+### Disable database features that require more disk space
+
+The following features require more disk space. You can disable or turn these features off to reclaim space on the nodes.
+
+- Time-to-live (TTL)
+- Point-in-time recovery (PITR)
+- xCluster
+- Change Data Capture (CDC)
+
+### Run manual compaction
+
+YugabyteDB automatically compacts data to keep the database running efficiently and reduce disk use. Certain operations, like bulk load of data or deleting a large number of rows, can cause a temporary spike in disk usage. If waiting for the automatic compaction task is not an option, you can manually run the compaction task using the following commands:
+
+- [Compact individual tables](../../../admin/yb-admin/#compact-table)
+- [Compact individual tablets](../../../admin/yb-ts-cli/#compact-tablet)
+
+Note that regardless of the above options, it is possible that some other activity on the node (such as remote bootstrap) can end up taking up more space and use up the space on the disk.
+
+The following sections describe the disk full scenarios and the recommended recovery actions.
+
+## Scenarios
+
+### The drive containing the log file is full
+
+During the tablet bootstrap that takes place during a YB-TServer restart, a new WAL file is allocated, irrespective of the amount of logs generated in the server's previous WAL file. If a YB-TServer has several tablets and results in a crash loop, each one of the tablets creates new WAL files on every bootstrap, filling up the disk. The process is unable to create new log files at tablet bootstrap time, and this can be identified by looking at the `/var/log/messages` files.
```

Comment:
You can remove "During the tablet bootstrap that takes place during a YB-TServer restart, a new WAL file is allocated, irrespective of the amount of logs generated in the server's previous WAL file. If a YB-TServer has several tablets and results in a crash loop, each one of the tablets creates new WAL files on every bootstrap, filling up the disk."